### ISSUE TITLE
Training accuracy

### ISSUE THREAD

**Issue by MAI-KOKO** [quotation2520/PG-RCNN#11]
Hi, I used single A6000 with batch_size=8 and epochs=100 for training, and the training accuracy on val dataset is one or two points lower than the paper, did you add any extra operation during training?
![微信图片_20240402083438](https://github.com/quotation2520/PG-RCNN/assets/82523570/7955bf72-8f12-4243-96b8-15b8c5f3ca2f)
![微信图片_20240402083449](https://github.com/quotation2520/PG-RCNN/assets/82523570/76ca763e-a89a-4643-919e-0103b5143c24)
If you have seen this question, please reply to me or contact my email: M778675961@163.com

**Comment by quotation2520** [comment]
Hi, we did not introduce any extra operation that is not introduced in the paper. 

We used batch size=16 and trained for 80 epochs.
If you changed the batch size, I'm pretty positive that you should also modify the learning rate accordingly.

Thanks!

**Comment by MAI-KOKO** [comment]
> Hi, we did not introduce any extra operation that is not introduced in the paper.
> 
> We used batch size=16 and trained for 80 epochs. If you changed the batch size, I'm pretty positive that you should also modify the learning rate accordingly.
> 
> Thanks!

I used 4 NVIDIA RTX 3090 GPUs with bs=16 and epochs=80 for training as per your paper's configuration. and the effect improved, but this problem still exists:
![67c5b02794ec7522478b778196e5b51](https://github.com/quotation2520/PG-RCNN/assets/82523570/79461954-5cd4-41e3-91b2-31c3116016ac)
![529102d5ad46f6952bebd4a6d91da19](https://github.com/quotation2520/PG-RCNN/assets/82523570/0a1656c4-e339-4300-af00-f095d042a16a)
![116f9e666ea5fc5ee960b5396f654b6](https://github.com/quotation2520/PG-RCNN/assets/82523570/60b7a327-f5d3-4e6a-a95c-e7b9e97c20f5)
I'm wondering if it's in the optimizer:
![c2415be9252238f4e331e4d67b5a865](https://github.com/quotation2520/PG-RCNN/assets/82523570/547b9006-64f1-4409-a032-cfc925d75fed)
It is related to whether BATCH_SIZE_PER_GPU should be set to 4 or for other reasons. I hope to get your reply or contact my email. Thank you very much.

**Comment by quotation2520** [comment]
Yes, if you use 4 GPU, you need to set BATCH_SIZE_PER_GPU = 4 to train with batch size=16.

Thanks!

**Comment by MAI-KOKO** [comment]
> Yes, if you use 4 GPU, you need to set BATCH_SIZE_PER_GPU = 4 to train with batch size=16.
> 
> Thanks!

I set BATCH_SIZE_PER_GPU to 4 and it came close to the thesis standard on the car and Pedestrian categories but dropped on cyclist, I was wondering how much you set it to when training with 4 Nvidia 3090

**Comment by quotation2520** [comment]
I set BATCH_SIZE_PER_GPU=4. 
Perhaps you can modify CAT_THRESH (category-wise threshold) to finetune the performance. 
Also note that performance can vary in different runs.

Thanks!
